modules/op-resolver-git-override-scm.adoc

```diff
@@ -0,0 +1,75 @@
+// This module is included in the following assembly:
+//
+// openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+= Specifying a remote pipeline or task using the Git resolver with the authenticated SCM API overriding the Git resolver configuration
+
+When creating a pipeline run, you can specify a remote pipeline from a Git repository using the authenticated SCM API. When creating a pipeline or a task run, you can specify a remote task from a Git repository. You can override the initial configuration settings in specific pipeline runs or tasks to customize the behavior according to different use cases.
+
+The following example task run references a remote task from a Git repository that overrides the previous resolver configuration:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: git-task-reference-demo
+spec:
+  taskRef:
+    resolver: git
+    params:
+    - name: org
+      value: tektoncd
+    - name: repo
+      value: catalog
+    - name: revision
+      value: main
+    - name: pathInRepo
+      value: task/git-clone/0.6/git-clone.yaml
+    - name: token
+      value: my-secret-token
+    - name: tokenKey
+      value: token
+    - name: scmType
+      value: github
+    - name: serverURL
+      value: https://ghe.mycompany.com
+----
++
+.Supported parameters to override the Git resolver
+|===
+| Parameter | Description | Example value
+
+| org
+| The organization for the repository.
+| tektoncd
```

Comment:
Overall, this table should be marked-up/formatted (for file names, field/value names, etc.) to better match the earlier tables, too. For example in this first row (though, apply uniformly throughout the table):
```suggestion
.Supported parameters to override the Git resolver
|===
| Parameter | Description | Example value

| `org`
| The organization for the repository.
| `tektoncd`
```